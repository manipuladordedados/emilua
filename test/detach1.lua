-- This test ensures that fibers whose joinable fiber handles got collected
-- will have their stacktrace printed to stderr (if err'ed)

collectgarbage("stop")

spawn(function()
    print('secondary fiber starts')
    error('tag', 0)
end)

print('secondary fiber spawned')
this_fiber.yield()
print('end of main fiber')
