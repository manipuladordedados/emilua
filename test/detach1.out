secondary fiber spawned
secondary fiber starts
end of main fiber
Fiber 0x1 panicked: 'tag'
stack traceback:
	input.lua:10: in function <input.lua:8>
